pom.xml

```diff
@@ -484,5 +502,80 @@
                 </plugins>
             </build>
         </profile>
+        <!-- Test against Bootable JAR -->
+        <profile>
+            <id>bootablejar.profile</id>
+            <activation>
+                <property>
+                    <name>ts.bootable</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-dependency-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>unpack-jboss-as</id>
+                                <goals>
+                                    <goal>unpack</goal>
+                                </goals>
+                                <phase>none</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <!-- Disable downloaded WildFly distribution filesystem setup -->
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>rename-unpacked-jboss-as</id>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                                <phase>none</phase>
+                            </execution>
+                            <execution>
+                                <id>initialize-basedirs</id>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                                <phase>none</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+            <dependencyManagement>
+                <dependencies>
+                    <!-- Bootable JAR related dependencies -->
+                    <dependency>
+                        <groupId>org.wildfly.arquillian</groupId>
+                        <artifactId>wildfly-arquillian-container-bootable</artifactId>
```

Comment:
What about to add this to dependencies directly (not to dependencyManagement)? It seems that this dependency is used in all modules relevant modules. In this case, this dependency don't need to be defined in each MP sub-modules in bootable profile.

Reply:
Eh, as said above this would me for this to be shaded into the `tooling-*` modules as well, although those are not tested (and IMO shouldn't be) when executing the Bootable JAR profile. WDYT?

Reply:
@fabiobrz The dependencies are specified per-profile. Wouldn't this mean they are needed only when the profile is activated?

Reply:
Thanks @honza-kasik - I now get what you and @marekkopecky mean. Iìve fixed it, let me know what you think.

Reply:
@fabiobrz Looks good, thank you! I guess it works as usual too, correct?

Reply:
@honza-kasik - looks like it's working, yep, see https://eap-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/eap-7.x-microprofile-testsuite-bootable/67/